lua/shipwright/transform/contrib/windows_terminal.lua

```diff
@@ -0,0 +1,85 @@
+--- windows terminal transform, expects a table in the shape:
+--
+-- @param colors {
+-- "name" : "Theme Name",
+
+-- "cursorColor": "#000000",
+-- "selectionBackground": "#000000",
```

Comment:
Need to be `cursor_color` and `selection_background`, those are the actual keys used in the template substitution.